stdlib/xmlrpc/client.pyi

```diff
@@ -13,9 +13,11 @@ class _SupportsTimeTuple(Protocol):
     def timetuple(self) -> time.struct_time: ...
 
 _DateTimeComparable = DateTime | datetime | str | _SupportsTimeTuple
-_Marshallable = Union[None, bool, int, float, str, bytes, tuple[Any, ...], list[Any], dict[Any, Any], datetime, DateTime, Binary]
-_XMLDate = Union[int, datetime, tuple[int, ...], time.struct_time]
-_HostType = Union[tuple[str, dict[str, str]], str]
+_Marshallable = (
+    bool | int | float | str | bytes | None | tuple[Any, ...] | list[Any] | dict[Any, Any] | datetime | DateTime | Binary
+)
+_XMLDate = int | datetime | tuple[int, ...] | time.struct_time
+_HostType = Union[str, tuple[str, dict[str, str]]]
```

Comment:
Why has this one changed?

Reply:
I guess I moved the order around to see if it made a difference, then reverted back to Union wrong
```suggestion
_HostType = Union[tuple[str, dict[str, str]], str]
```